lib/middleware/jsonIDValidator.ts

```diff
@@ -84,57 +70,48 @@ export const layoutIDValidator = (options?: ValidateOptions): MiddlewareRequest
  */
 export const subElementsIDValidator = (options?: ValidateOptions): MiddlewareRequest => {
   return async (req: NextApiRequest, res: NextApiResponse): Promise<MiddlewareReturn> => {
-    try {
-      if (options?.runValidationIf?.(req) === false) {
-        return { next: true };
-      }
-      const formProperties: FormProperties = options?.jsonKey
-        ? req.body[options.jsonKey]
-        : req.body;
+    if (options?.runValidationIf?.(req) === false) {
```

Comment:
After looking at the code I did not find a reason why this would have to be wrapped in a `try/catch` block